cli/commands/project_sync.go

```diff
@@ -267,10 +295,10 @@ func (cmd *ProjectSync) WaitForUnisonContainer(containerName string, timeoutSeco
 
 // The local unison process is finished initializing when the log file exists
 // and has stopped growing in size
-func (cmd *ProjectSync) WaitForSyncInit(logFile string, timeoutSeconds int, syncWaitSeconds int) {
+func (cmd *ProjectSync) WaitForSyncInit(logFile string, workingDir string, timeoutSeconds int, syncWaitSeconds int) {
 	cmd.out.Info.Print("Waiting for initial sync detection")
 
-	var tempFile = fmt.Sprintf(".%s.tmp", logFile)
+	var tempFile = filepath.Join(workingDir, fmt.Sprintf(".test-sync-start.tmp"))
```

Comment:
Don;t need a Sprintf here is the log file name is not patterned from a variable. `%s` or whatevs.

Reply:
I think what I meant to say is, why are you not using `logFile`

Reply:
Also, `logFile` should already be a full absolute path, so I'm thinking that the `filepath.Join` is not needed.

Reply:
This logic predates the PR. I think logFile is used by unison, and we're checking that tempFile is synced. I wonder what happens if a project ignores *.tmp files... might need a parameter or less fraught ext.